src/components/ModalFlows/CreateProposalScreens/ActionFees.js

```diff
@@ -51,98 +51,56 @@ function ActionFeesModal({ agreement, onCreateTransaction }) {
         </span>
       </Field>
 
-      {compactMode && (
-        <div
-          css={`
-            margin-top: ${GU * 2}px;
-            weight: 100%;
-          `}
-        >
-          <img
-            css={`
-              height: ${GU * 4}px;
-              width: ${GU * 4}px;
-            `}
-            src={iconFees}
-            alt=""
-          />
-        </div>
-      )}
-
       <div
         css={`
           display: flex;
           width: 100%;
           height: ${GU * 4}px;
           margin-top: ${!compactMode ? GU * 3 : 0}px;
+          align-items: center;
         `}
       >
-        {!compactMode && (
-          <div
-            css={`
-              flex: 0.25;
-            `}
-          >
-            <img
-              css={`
-                height: ${GU * 4}px;
-                width: ${GU * 4}px;
-              `}
-              src={iconFees}
-              alt=""
-            />
-          </div>
-        )}
+        <img
+          css={`
+            height: ${GU * 4}px;
+            width: ${GU * 4}px;
+          `}
+          src={iconFees}
```

Comment:
Lets use the `img` properties 

```suggestion
          css={`
            height: ${GU * 4}px;
            width: ${GU * 4}px;
          `}
          src={iconFees}
          height={4 * GU}
          width={4 * GU}
```